velox/type/DecimalUtils.cpp

```diff
@@ -105,4 +105,9 @@ std::string DecimalUtil::toString<ShortDecimal>(
   auto decimalType = type->asShortDecimal();
   return formatDecimal(decimalType.scale(), value.unscaledValue());
 }
+
+template <typename T>
+std::string DecimalUtil::toString(const T& value, const TypePtr& type) {
```

Comment:
this is not needed.

Reply:
This is required. I get errors at the time of integration in SimpleVector.h.

Reply:
What is the error?